src/main/java/io/openepcis/convert/Conversion.java

```diff
@@ -0,0 +1,90 @@
+package io.openepcis.convert;
+
+import io.openepcis.constants.EPCISFormat;
+import io.openepcis.constants.EPCISVersion;
+import lombok.Getter;
+
+@Getter
+public class Conversion {
+
+    private EPCISFormat fromMediaType;
+    private EPCISVersion fromVersion;
+    private EPCISFormat toMediaType;
+    private EPCISVersion toVersion;
+    private boolean generateGS1CompliantDocument;
+
+    private Conversion() {
+    }
+
+    public static FromMediaTypeStage newBuilder() {
```

Comment:
rename method to just builder() - which is more common. Of course it creates a new instance for Builder ;-)
make sure to setup meaningful defaults